src/devices/MotorHat/MotorHat.cs

```diff
@@ -0,0 +1,193 @@
+        /// </remarks>
+        public MotorHat() : this(new I2cConnectionSettings(1, I2cAddressBase + 0b000000), 1600)
+        {
+            //Use the following code to set an address equivalent to the one configured in your device jumpers
+            //var busId = 1;
+            //var selectedI2cAddress = 0b000000;     // A5 A4 A3 A2 A1 A0
+            //var deviceAddress = Pca9685.I2cAddressBase + selectedI2cAddress;
+            //var settings = new I2cConnectionSettings(busId, deviceAddress);
+        }
+
+        /// <summary>
+        /// Creates a <see cref="DCMotor"/> object for the specified channel.
+        /// </summary>
+        /// <param name="motorNumber">A motor number from 1 to 4.</param>
+        /// <remarks>
+        /// The motorNumber parameter refers to the motor numbers M1, M2, M3 or M4 printed in the device.
+        /// </remarks>
+        public DCMotor.DCMotor CreateDCMotor(byte motorNumber)
+        {
+            if (motorNumber < 1 || motorNumber > 4)
+                throw new ArgumentOutOfRangeException(nameof(motorNumber), $"Must be between 1 and 4, corresponding with M1, M2, M3 and M4. (Received: {motorNumber}");
+
+            // The PCA9685 PWM controller is used to control the inputs of two dual motor drivers.
+            // These correspond to motor hat screw terminals M1, M2, M3 and M4.
+            // Each motor driver circuit has one speed pin and two IN pins.
+            // The PWM pin expects a PWM input signal. The two IN pins expect a logic 0 or 1 input signal.
+            // The variables speed, in1 and in2 variables identify which PCA9685 PWM output pins will be used to drive this DCMotor.
+            // The speed variable identifies which PCA9685 output pin is used to drive the PWM input on the motor driver.
+            // And the in1 and in2 variables are used to specify which PCA9685 output pins are used to drive the xIN1 and xIN2 input pins of the motor driver.
+
+            byte speedPin, in1Pin, in2Pin;
+
+            switch (motorNumber)
+            {
+                case 1:
+                    speedPin = 8;
+                    in2Pin = 9;
+                    in1Pin = 10;
+                    break;
+                case 2:
+                    speedPin = 13;
+                    in2Pin = 12;
+                    in1Pin = 11;
+                    break;
+                case 3:
+                    speedPin = 2;
+                    in2Pin = 3;
+                    in1Pin = 4;
+                    break;
+                case 4:
+                    speedPin = 7;
+                    in2Pin = 6;
+                    in1Pin = 5;
+                    break;
+                default:
+                    throw new ArgumentException($"MotorHat Motor must be between 1 and 4 inclusive. Received: {motorNumber}");
+            }
+
+            //Add to the channelsUsed list, the channels we will use for this DCMotor
```

Comment:
space after `//`

Reply:
btw no need to comment at all, comment above `channelsUsed` is enough (no comment is also fine